.github/workflows/__cmake-format-check.yaml

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: (C) The kokkos-fft development team, see COPYRIGHT.md file
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
+
+name: cmake-format check
+
+on:
+  workflow_call:
+
+permissions: read-all
+
+jobs:
+  cmake-formatting-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+    - name: cmake-format lint action
+      uses: puneetmatharu/cmake-format-lint-action@2ac346e79e7ceac958bc637c1391285fb335ed7c # v1.0.5
+      with:
+        args: --config-files .cmake-format.py --in-place
+        file-regex: '(.*\.cmake$|.*\.cmake\.in$|CMakeLists.txt$)'
+    - name: check
+      run: git diff --exit-code 
```

Comment:
Why the option `--check` is not enough ?

Reply:
It does modifications in place and we check the diff to the files in git to have printed output

Reply:
I would like to print the changes we need. 
Basically, the same as lint.

Reply:
Oh, Jacob already explained. 
Thank you for the comments

Reply:
ok thx